### ISSUE TITLE
Can't create conditional questions

### ISSUE THREAD

**Issue by ryooan** [Metaculus/metaculus#1029]
Currently am unable to create conditional questions. I'm experiencing several issues:

* When I try to put in a non-binary child, it doesn't make the question, just gives errors and loads forever.
* When I try to put in a binary child, it makes the question but grabs the wrong child. For example, I put in id [28830](https://www.metaculus.com/questions/28830/90-of-us-road-vehicles-sold-in-2075-have-level-5-autonomy/) and it instead makes [this conditional](https://www.metaculus.com/questions/28913/practice-will-joe-biden-be-elected-us-president-in-2024/) with this [other question](https://www.metaculus.com/questions/28878/) as the child, and clicking on the child gives this strange url, which looks like a group url but it's just a regular question: `https://www.metaculus.com/questions/28878/?sub-question=28830`
* Also it might not be letting me use a binary subquestion from a group question, like [this one](https://www.metaculus.com/questions/11245/2024-us-presidential-election-winner/?sub-question=5717), as the parent, though given the other issues I'm not able to tell if this is the case or not.

**Comment by ryooan** [comment]
Also mentioning a couple of related issues for conditionals, the above seems highest priority, but these others may be related.

https://github.com/Metaculus/metaculus/issues/1010

https://github.com/Metaculus/metaculus/issues/1013

**Comment by ryooan** [comment]
@lsabor bump on this one, if possible would be great to unblock conditional creation soon (totally understand though if other stuff is higher priority)

**Comment by ncarazon** [comment]
I’ve investigated the issues mentioned in this and related tickets, and I don't believe they can be fully resolved on the frontend alone.

- Fixed a bug with infinite loading on error.
- The behavior is quite strange: the correct question ID is passed, but the backend returns a different ID as a conditional with a subquestion link.
- It seems that we don’t currently support subquestions as conditional parent/child. To handle this on the frontend, I would need an endpoint to fetch data about subquestions to verify if they are valid type. I'm not sure about the required backend changes.
- Regarding editing, some existing and open questions can be edited without issue. However, I can’t edit unapproved questions or those where the parent/child is a subquestion. From the BE error message I can guess that there is an issue with an open_time field.
- For the modal, I made some changes, but it looks like Hlib refactored it slightly. But even after that, the open time defaults to the current time, even though I select a different timestamp to send.

**Comment by ncarazon** [comment]
I’ll remain assigned in case any frontend changes are required

**Comment by lsabor** [comment]
@ryooan ready for your QA

**Comment by ryooan** [comment]
@lsabor looks good, can create questions fine, only minor issue is that the preview of the question doesn't show up at first, it flashes suddenly when I click submit or will show up if I click the "+" button after adding a question link. But that's minor and not blocking anything, so if you want to track that elsewhere that's good with me!

**Comment by lsabor** [comment]
Oh, yeah that's intentional. It doesn't make the request until you press the "+" button so that it doesn't keep making a request every time you change a character. Maybe there's a better way to do the UI, but if it is important, we can make another ticket. I'm closing this one as complete.
